frontend/src/scenes/notebooks/Nodes/NotebookNodeBacklink.tsx

```diff
@@ -97,4 +95,49 @@ export const NotebookNodeBacklink = Node.create({
     addNodeView() {
         return ReactNodeViewRenderer(Component)
     },
+
+    addPasteRules() {
+        return [
```

Comment:
are these similar enough we could generate them so that the similarities/differences are more clear?

(e.g. nodePasteThingies.map(blah))